apps/studio/pages/project/[ref]/storage/analytics/buckets/[bucketId].tsx

```diff
@@ -20,13 +23,22 @@ const AnalyticsBucketPage: NextPageWithLayout = () => {
   if (!project || !projectRef) return null
 
   return (
-    <div className="storage-container flex flex-grow p-4">
+    <div className="storage-container flex flex-grow">
       {isError && <StorageBucketsError error={error as any} />}
 
       {isSuccess ? (
         !bucket ? (
           <div className="flex h-full w-full items-center justify-center">
-            <p className="text-sm text-foreground-light">Bucket {bucketId} cannot be found</p>
+            <Admonition
+              className="max-w-md"
+              type="default"
+              title="Unable to find bucket"
+              description={`${bucketId ? `The template "${bucketId}"` : 'This template'} doesn’t seem to exist.`}
+            >
+              <Button asChild type="default" className="mt-2">
+                <Link href={`/project/${projectRef}/storage/analytics`}>Head back</Link>
+              </Button>
+            </Admonition>
```

Comment:
Better empty states as discussed elsewhere. Documenting as a pattern for future reference.